seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-1/pom.xml

```diff
@@ -81,6 +81,11 @@
             <artifactId>postgresql</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.oceanbase</groupId>
+            <artifactId>oceanbase-client</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Is this necessary? Because oceanbase test only on `connector-jdbc-e2e-part-2`.

Reply:
I added it by mistake.